native/ex_tokenizers/scripts/install-cross-deps

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Script to install dependencies for cross compilation
+
+dpkg --add-architecture $CROSS_DEB_ARCH
+
+apt-get update
+apt-get install --assume-yes libssl-dev:$CROSS_DEB_ARCH openssl:$CROSS_DEB_ARCH
```

Comment:
@philss did you successfully build on `arm-unknown-linux-gnueabihf` and `x86_64-unknown-linux-musl`? When I tried that it failed for `gnueabihf` (I think because of https://github.com/cross-rs/cross/pull/1018/files), while with `musl` I don't think there's a package available (looking [here](https://packages.ubuntu.com/bionic/libssl-dev)).

Reply:
@jonatanklosko I could build for those targets locally. I'm trying to build using the CI.
Did you try to install using the "pre-build" command in the "Cross.toml" file? I also bumped cross version, so maybe this is the reason it worked for me.

Reply:
@jonatanklosko I think it should work good now.

Reply:
Yeah, vendoring definitely works, we can even make it such that this only happens on CI via feature flags:

```
[dependencies]
# ...
openssl = { version = "0.10", optional = true }

[features]
static_openssl = ["openssl/vendored"]
```

This does link openssl statically though.